<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestTopologyForm</class>
 <widget class="QWidget" name="TestTopologyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="but_netgearinfo">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Netgear Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_count">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_gateway">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_local">
       <property name="text">
        <string>showLocal</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_flags">
       <property name="text">
        <string>showFlags</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_netbios">
       <property name="text">
        <string>showNetbios</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_upnp">
       <property name="text">
        <string>showUpnp</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_currentsetting">
       <property name="text">
        <string>showCurrentSetting</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_attachdevice">
       <property name="text">
        <string>showAttachDevice</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_fupdate">
       <property name="text">
        <string>showFupdate</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_all">
       <property name="text">
        <string>showAll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="but_last">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_next">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_start">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_count">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>111</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>subscrib</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_attachdevice</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_currentsetting</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_upnp</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_netbios</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_flags</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_local</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBox_fupdate</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
